.gitignore

```diff
@@ -55,6 +55,7 @@ bld/
 
 # Visual Studio 2017 auto generated files
 Generated\ Files/
+**/*.Properties/launchSettings.json
```

Comment:
I think having the . after the * will cause it not to match to the launch settings

Reply:
I was copying the syntax used later with
**/*.DesktopClient/ModelManifest.xml
Is that not valid here?

Reply:
I am not 100% sure, it may be. One way to find out would be to readd the launchsettings file in the same location and see if it is ignored.

Reply:
Seems I did need to remove the dot.  Not sure if the reference I used was also invalid, but I'll leave it for now.